src/coreclr/jit/llvmlower.cpp

```diff
@@ -997,36 +974,31 @@ void Llvm::failUnsupportedCalls(GenTreeCall* callNode)
         failFunctionCompilation();
     }
 
-    if (callNode->gtCallArgs != nullptr)
+    for (CallArg& callArg : callNode->gtArgs.Args())
     {
-        for (GenTree* operand : callNode->Operands())
+        GenTree* operand = callArg.GetNode();
```

Comment:
I think the whole:
```
        GenTree* operand = callArg.GetNode();
        if (operand->IsArgPlaceHolderNode() || !operand->IsValue())
        {
            // Either of these situations may happen with calls.
            continue;
        }
        if (operand == callNode->gtControlExpr || operand == callNode->gtCallAddr)
        {
            // vtable target or indirect target
            continue;
        }
```
Can just be deleted as it's a no-op. We only care about blocking VSD.

Reply:
Indeed.